### PR TITLE
added crate of the week, notable PRs + clippy workshop

### PR DESCRIPTION
**Pull request by llogiq**
Since we were asked to do this stuff in the open, I'll just leave this PR here. :smile:

### CODE REVIEW

File: drafts/2016-03-14-this-week-in-rust.md [hunk 1]
```diff
@@ -53,12 +54,25 @@ If you are a Rust project owner and are looking for contributors, please submit
...
 ## Notable changes
 
+* [implement RFC 1192 inclusive ranges](https://github.com/rust-lang/rust/pull/30884)
+* [SIMD intrinsics for fused multiply-add](https://github.com/rust-lang/rust/pull/32066)
+* [AVX broadcast/conversion intrinsics](https://github.com/rust-lang/rust/pull/32140)
+* [New i586-pc-windows-msvc target](https://github.com/rust-lang/rust/pull/32034)
+* [Huge-String-Slice-Overflow-DoS averted](https://github.com/rust-lang/rust/pull/32064)
+* [Forbid having overlapping fns in inherent impls for same type](https://github.com/rust-lang/rust/pull/31925)
+* [Add Pass manager for MIR](https://github.com/rust-lang/rust/pull/31916)
+* [fn item type shenanigans](https://github.com/rust-lang/rust/pull/31710)
```
Comment (bluss) [inline drafts/2016-03-14-this-week-in-rust.md]: Could be renamed - *fn item types are properly unique and have size zero*.

This PR is interesting too Add impl of FnOnce to AssertRecoverSafe https://github.com/rust-lang/rust/pull/32102

Comment (llogiq) [inline drafts/2016-03-14-this-week-in-rust.md]: Thanks!

**Comment by nasa42** [comment]
Thanks @llogiq! PR is definitely better. Appreciate it!
One small suggestion:
Please replace "The crate of this week is" with "This week's Crate of the Week is". I'll manually change for this issue.

File: drafts/2016-03-14-this-week-in-rust.md (hunk 1 above)
Comment (nasa42) [inline drafts/2016-03-14-this-week-in-rust.md]: I've amended these changes. Thanks @bluss!

**Comment by llogiq** [comment]
Ok, thank you @nasa42

**Pull request merged**